.github/workflows/tests.yaml

```diff
@@ -76,6 +86,98 @@ jobs:
                   path: |
                       .environment/gn_out/.ninja_log
                       .environment/pigweed-venv/*.log
+            - name: Validate that xml are parseable
+              timeout-minutes: 10
+              # The sub-items being run here are the same as the input XMLs listed
+              # at src/app/zap-templates/zcl/zcl.json
+              #
+              # This ensures that the syntax of the XML can always be parsed/validated, however it
+              # does not enforce that the content is understood (that part is covered by parser
+              # unit tests)
+              #
+              # Also note ordering: global attributes are parsed first because other XML files may
+              # reference these. 
+              #
+              # TODO: some multipass/postprocess may be more appropriate to allow any ordering of 
+              #       things.
+              run: |
+                  ./scripts/run_in_build_env.sh \
+                    "./scripts/idl/xml_parser.py \
+                      --no-print \
+                      --log-level info \
+                      src/app/zap-templates/zcl/data-model/chip/global-attributes.xml \
```

Comment:
Could this be recast to store the list of xml paths in a file and pipe that into the script?
Anything that makes running the CI command locally more user friendly would be good.

Reply:
I considered that, however I would like the parser tests to be explicitly independent from the zap input it uses.

The benefit of this parsing is that we can validate xmls and we can eventually do codegen without involving zap.